2019_03_14_ChangeStringTask/src/Main.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
```

Comment:
Do not afraid to create a new branch from master before starting with the new task.